nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/main/java/org/nuxeo/ecm/automation/core/mail/Mailer.java

```diff
@@ -97,12 +98,12 @@ public Mailer(String sessionName, Properties config) {
         final String user = config.getProperty("mail.smtp.user");
         final String pass = config.getProperty("mail.smtp.password");
         if (user != null && pass != null) {
-            setAuthenticator(new Authenticator() {
+            this.auth = new Authenticator() {
                 @Override
                 protected PasswordAuthentication getPasswordAuthentication() {
                     return new PasswordAuthentication(user, pass);
                 }
-            });
+            };
```

Comment:
i think we can use `this.auth = new org.nuxeo.mail.MailAuthenticator(config);` instead of the anonymous class wdyt